run_endpoint.sh

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env bash
+
+# Set up the routing needed for the simulation
+/setup.sh
+
+if [ -n "$TESTCASE" ]; then
+    case "$TESTCASE" in
+    # TODO: add supported test cases here
+    "versionnegotiation"|"handshake"|"transfer"|"retry"|"resumption"|\
+    "multiconnect"|"zerortt"|"chacha20")
+        ;;
+    *)
+        exit 127
+        ;;
+    esac
+fi
+
+# The following variables are available for use:
+# - ROLE contains the role of this execution context, client or server
+# - SERVER_PARAMS contains user-supplied command line parameters
+# - CLIENT_PARAMS contains user-supplied command line parameters
+
+if [ "$ROLE" == "client" ]; then
+    # Wait for the simulator to start up.
+    /wait-for-it.sh sim:57832 -s -t 30
+    cd /downloads || exit
+
+    # TODO: add client support
+    # I am not sure if the msquic codebase has an h09 client?
+    exit 127
+
+elif [ "$ROLE" == "server" ]; then
+    case "$TESTCASE" in
+    "retry")
+        SERVER_PARAMS="-retry:1 $SERVER_PARAMS"
+        ;;
+    *)
+        ;;
+    esac
+
+    quicinteropserver ${SERVER_PARAMS} -root:/www -listen:* -port:443 \
+        -file:/server.crt -key:/server.key 2>&1
```

Comment:
Would it be possible to somehow dynamically generate these certs instead of checking them in? I'd prefer not to have checked in certs? Another option is to add a `-selfsign` option to quicinteropserver. We already have a helper function (`QuicPlatGetSelfSignedCert`) that could be used to easily add that functionality.

Reply:
Fixed in latest push